test/python/test_docker_pack.py

```diff
@@ -244,7 +237,7 @@ def test_docker_e2e(project, docker_image, tmpdir, docker_client):
         image_name,
         environment=environment,
         ports={'8082': '8082'},
-        name='{}-instance-1'.format(project['name']),
```

Comment:
Why do you remove a `project` from this test?

Reply:
To simplify the test.

Reply:
The project is created on `docker_image` fixture setup.

Reply:
May be pass `project name` with image name to eliminate such hardcode?

Reply:
Yes, I think that docker_image object can contain field `project_name`.